.github/workflows/olm_tests.yaml

```diff
@@ -122,80 +122,83 @@ jobs:
-          timeout 60 bash -c 'until [[ $(kubectl get deployment/codeflare-operator-manager -n '${{ env.SUBSCRIPTION_NAMESPACE }}') ]]; do sleep 5 && echo "$(kubectl get deployment/codeflare-operator-manager -n '${{ env.SUBSCRIPTION_NAMESPACE }}')"; done'
-          kubectl wait -n ${{ env.SUBSCRIPTION_NAMESPACE }} deployment/codeflare-operator-manager --for=condition=Available=true --timeout=60s
-
-          echo Checking that correct CSV is available
-          CSV_VERSION=$(kubectl get ClusterServiceVersion/codeflare-operator.${VERSION} -n openshift-operators -o json | jq -r .spec.version)
-          if [ "v${CSV_VERSION}" != "${VERSION}" ]; then
-            echo "CSV version v${CSV_VERSION} doesn't match expected version ${VERSION}"
-            exit 1
-          fi
-        env:
-          CATALOG_IMG: "${{ env.REGISTRY_ADDRESS }}/codeflare-operator-catalog:v0.0.1"
-          SUBSCRIPTION_NAME: "codeflare-operator"
-          SUBSCRIPTION_NAMESPACE: "openshift-operators"
-
-      - name: Run OLM Upgrade e2e Appwrapper Job status test to monitor training
-        run: |
-          export CODEFLARE_TEST_OUTPUT_DIR=${{ env.TEMP_DIR }}
-          echo "CODEFLARE_TEST_OUTPUT_DIR=${CODEFLARE_TEST_OUTPUT_DIR}" >> $GITHUB_ENV
-          set -euo pipefail
-          go test -timeout 30m -v ./test/upgrade -run TestMNISTCheckAppWrapperStatus -json 2>&1 | tee ${CODEFLARE_TEST_OUTPUT_DIR}/gotest.log | gotestfmt
-
-      - name: Run e2e tests against built operator
-        run: |
-          export CODEFLARE_TEST_OUTPUT_DIR=${{ env.TEMP_DIR }}
-          echo "CODEFLARE_TEST_OUTPUT_DIR=${CODEFLARE_TEST_OUTPUT_DIR}" >> $GITHUB_ENV
-
-          set -euo pipefail
-          go test -timeout 30m -v ./test/e2e -json 2>&1 | tee ${CODEFLARE_TEST_OUTPUT_DIR}/gotest.log | gotestfmt
-
-      - name: Print CodeFlare operator logs
-        if: always() && steps.deploy.outcome == 'success'
-        run: |
-          echo "Printing CodeFlare operator logs"
-          kubectl logs -n openshift-operators --tail -1 -l app.kubernetes.io/name=codeflare-operator | tee ${CODEFLARE_TEST_OUTPUT_DIR}/codeflare-operator.log
-
-      - name: Print KubeRay operator logs
-        if: always() && steps.deploy.outcome == 'success'
-        run: |
-          echo "Printing KubeRay operator logs"
-          kubectl logs -n ray-system --tail -1 -l app.kubernetes.io/name=kuberay | tee ${CODEFLARE_TEST_OUTPUT_DIR}/kuberay.log
-
-      - name: Export all KinD pod logs
-        uses: ./common/github-actions/kind-export-logs
-        if: always() && steps.deploy.outcome == 'success'
-        with:
-          output-directory: ${CODEFLARE_TEST_OUTPUT_DIR}
-
-      - name: Upload logs
-        uses: actions/upload-artifact@v4
-        if: always() && steps.deploy.outcome == 'success'
-        with:
-          name: logs
-          retention-days: 10
-          path: |
-            ${{ env.CODEFLARE_TEST_OUTPUT_DIR }}/**/*.log
+      # TODO: Disabled during the MCAD to Kueue transition.
```

Comment:
We'll need a release of the codeflare operator with the new code.  After that is done, we should be able to re-enable.

Reply:
I think we disable the workflow manually instead of introducing those changes. Alternatively I think that's acceptable we force merge the PR with the OLM workflow failing.

Reply:
ok.  I'll revert the changes to this workflow and assume we'll ignore/override the failure to merge the PR.

Reply:
kept the change to go version; reverted the rest of the file